packages/backend-common/package.json

```diff
@@ -57,6 +57,7 @@
     "knex": "^0.95.1",
     "lodash": "^4.17.21",
     "logform": "^2.1.1",
+    "micromatch": "^4.0.4",
```

Comment:
minimatch doesn’t work as expected, on `http://` treat `//` as a single `/`
ref:micromatch/micromatch#89

Reply:
It does? O_o, seems to work fine for origin matching here: https://github.com/backstage/backstage/blob/7d3228fefb2bca9712dedb3839bb2119c6ccb88a/plugins/auth-backend/src/service/router.ts#L177

Reply:
hmm, seems to be a problem with `makeRe` function https://github.com/isaacs/minimatch/issues/29

I was looking to generate regex to be managed by cors directly, but don’t see any difference with the auth-backend approach, so I will change this to generate a function instead and remove the new dependency.